src/pages/sujeongKim/Main/Main.js

```diff
@@ -1,8 +1,94 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './Main.scss';
+import FeedItem from './FeedItem';
+import StoryItem from './StoryItem';
+import Recommend from './Recommend';
 
 const MainSujeong = () => {
-  return <div>Main</div>;
+  const [feedItem, setFeedItem] = useState([]);
+  const [storyItem, setStoryItem] = useState([]);
+  const [recommendItem, setRecommendItem] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/feed_data.json')
+      .then(res => res.json())
+      .then(data => {
+        setFeedItem(data);
+      });
+    fetch('/data/story_data.json')
+      .then(res => res.json())
+      .then(data => {
+        setStoryItem(data);
+      });
+    fetch('/data/recommend_data.json')
+      .then(res => res.json())
+      .then(data => {
+        setRecommendItem(data);
+      });
+  }, []);
+
+  return (
+    <React.Fragment>
+      <nav className="nav">
+        <div className="logoGroup">
+          <img
+            src="/images/sujeongKim/001-instagram.png"
+            alt="westagram 로고 이미지"
+          />
+          <h1 className="logo">westagram</h1>
+        </div>
+        <input
+          type="text"
+          className="search"
+          name="search"
+          placeholder="검색"
+        />
+        <div className="navRight">
+          <div className="exploreIcon"> </div>
+          <div className="heartIcon"></div>
+          <div className="profileIcon"></div>
```

Comment:
```suggestion
          <div className="exploreIcon" />
          <div className="heartIcon" />
          <div className="profileIcon" />
```

self closing으로 해주세요!